frontend/packages/console-shared/src/utils/__tests__/icon-utils.spec.ts

```diff
@@ -0,0 +1,17 @@
+import * as _ from 'lodash';
+import { sampleClusterServiceVersions } from '@console/dev-console/src/components/topology/__tests__/topology-test-data';
+import * as operatorLogo from '../../images/operator.svg';
+import { getImageForCSVIcon } from '../icon-utils';
+import { mockCSVIcon } from '../__mocks__/mock-csv-icon';
+
+describe('Icon Utils', () => {
+  it('should return icon from csv data', () => {
+    const mockCSV = _.cloneDeep(sampleClusterServiceVersions.data[0]);
+    const icon = _.get(mockCSV, 'spec.icon.0');
```

Comment:
You don't need topology test data for this unit test.
Please update to use an object that matches type `CSVIcon`.

Looks like there is a topology specific test added later on that captures this same data flow.

Reply:
Done.